com.github.benini.scid.yml

```diff
@@ -0,0 +1,122 @@
+app-id: com.github.benini.scid
+branch: stable
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: /app/scid/scid
```

Comment:
the command should be `/app/bin` and the command property a command name, not a path.

Reply:
The binary need to be in /app/scid

Reply:
symlink, wrapper script. etc. Many solutions are available to work around broken software.

Reply:
oh, that's rude

Reply:
It's not strictly required but definitely a good practice.